docs/fundamentals/code-analysis/quality-rules/ca1000.md

```diff
@@ -22,6 +22,7 @@ dev_langs:
 | **Rule ID**                         | CA1000                       |
 | **Category**                        | [Design](design-warnings.md) |
 | **Fix is breaking or non-breaking** | Breaking                     |
+| **Enabled by default in .NET 7**    | No                           |
```

Comment:
Are these changes automated?

Reply:
The table formatting is done using a VS Code extension (per file). The rest is all manual.